bin/ajax

```diff
@@ -81,41 +80,44 @@ def main_ajax():
     if args.number:
         remove_run_from_host(args.number, delete_live=args.delete_live, force=args.force)
 
-    if args.mode:
-        if args.mode == 'clean_ceph':
-            clean_cehp()
-        elif args.mode == 'clean_high_level_data':
-            clean_high_level_data()
-        elif args.mode == 'clean_non_latest':
-            clean_non_latest()
-        elif args.mode == 'clean_unregistered':
-            clean_unregistered()
-            if hostname == eb_can_clean_ceph:
-                clean_unregistered(delete_live=True)
-        elif args.mode == 'clean_raw_records_nv':
-            clean_raw_records_nv()
-        elif args.mode == 'clean_abandoned':
-            clean_abandoned()
-            if hostname == eb_can_clean_ceph:
-                clean_abandoned(delete_live=True)
-        elif args.mode == 'clean_all':
+    if args.clean:
+        _modes = {'ceph': clean_cehp,
+                  'high_level_data': clean_high_level_data,
+                  'non_latest': clean_non_latest,
+                  'unregistered': clean_unregistered,
+                  'raw_records_nv': clean_raw_records_nv,
+                  'abandoned': clean_abandoned,
+                  'old_hash': clean_old_hash,
+                  'database': clean_database}
+
+        if args.clean in _modes:
+            # Do this mode
+            _modes[args.clean]()
+            if hostname == eb_can_clean_ceph and (
+                    args.clean in ['abandoned', 'unregistered', 'database']):
+                _modes[args.clean](delete_live=True)
+
+            log.info(f'Done with {args.clean}')
+        elif args.clean == 'all':
             while True:
                 if hostname == eb_can_clean_ceph:
                     clean_cehp()
                     clean_abandoned(delete_live=True)
                     clean_unregistered(delete_live=True)
+                    clean_database(delete_live=True)
 
                 clean_high_level_data()
                 clean_unregistered()
                 clean_raw_records_nv()
                 clean_abandoned()
                 clean_non_latest()
+                clean_database()
                 if not args.execute:
                     break
                 log.info(f'Loop finished, take a {ajax_thresholds["nap_time"]} s nap')
                 time.sleep(ajax_thresholds['nap_time'])
         else:
-            raise ValueError(f'Unknown mode {args.mode}')
+            raise ValueError(f'Unknown cleaning mode {args.clean}')
 
 
 def clean_cehp():

```

Comment:
`:%s/clean_cehp/clean_ceph/gc` ?

Reply:
Sorry I'm not sure if I follow this comment

Reply:
It's the vim command to replace `clean_cehp` with `clean_ceph`.

Reply:
thx done!